.github/workflows/ci.yml

```
@@ -17,37 +17,37 @@ jobs:
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 8.x
+
+    - name: Run the Cake script
+      uses: cake-build/cake-action@v1
+      with:
+        verbosity: Diagnostic
+
+    - uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)